pynucastro/rates/library.py

```diff
@@ -701,15 +702,15 @@ def __init__(self):
 class TabularLibrary(Library):
     """Load all of the tabular rates known and return a Library"""
 
-    lib_path = f"{os.path.dirname(__file__)}/../library/tabular"
+    lib_path = Path(__file__).resolve().parents[1]/"library"/"tabular"
```

Comment:
I think this is easier to read without the extra quoting and `resolve()` call (that should only be needed when dealing with symlinks or making a relative path absolute, neither of which apply here):
```suggestion
    lib_path = Path(__file__).parents[1]/"library/tabular"
```
Same for the subclasses below.